### ISSUE TITLE
EPERM: Operation not permitted, unlink '<app_name>/app/client'

### ISSUE THREAD

**Issue by chrisvoncsefalvay** [iron-meteor/iron-cli#242]
Potential duplicate of #240.

```
created mself
created mself/.iron
created mself/.iron/config.json
created mself/bin
created mself/bin/.gitkeep
created mself/build
created mself/build/.gitkeep
created mself/build/README
created mself/config
created mself/config/.gitkeep
created mself/config/development
created mself/config/development/env.sh
created mself/config/development/settings.json
created mself/config/production
created mself/config/production/env.sh
created mself/config/production/settings.json
Creating project  app... -
/usr/local/lib/node_modules/iron-meteor/node_modules/fibers/future.js:267
						throw(ex);
						^

Error: EPERM: operation not permitted, unlink 'mself/app/client'
    at Error (native)
    at Object.fs.unlinkSync (fs.js:980:18)
    at /usr/local/lib/node_modules/iron-meteor/lib/tools/meteor.js:35:10
    at Function._.each._.forEach (/usr/local/lib/node_modules/iron-meteor/node_modules/underscore/underscore.js:153:9)
    at Command.createEmptyMeteorProject (/usr/local/lib/node_modules/iron-meteor/lib/tools/meteor.js:32:7)
    at /usr/local/lib/node_modules/iron-meteor/lib/commands/init.js:63:10
    at Object.CurrentConfig.withValue (/usr/local/lib/node_modules/iron-meteor/lib/config.js:65:13)
    at Command.<anonymous> (/usr/local/lib/node_modules/iron-meteor/lib/commands/init.js:58:24)
    at Command.invoke (/usr/local/lib/node_modules/iron-meteor/lib/command.js:111:26)
    at Command.<anonymous> (/usr/local/lib/node_modules/iron-meteor/lib/commands/create.js:15:37)
```

The workaround in #240 works great, so this might be the same issue, but as the description there was patchy, I was not entirely sure. Close the issue w/my apologies if it's a duplicate.

**Comment by Kosmin** [comment]
:+1:

**Comment by chrisbutler** [comment]
should be fixed in 1.5.3
